src/html/shared/header.html

```diff
@@ -1,23 +1,21 @@
-<header class="c-header">
+<nav class="c-header  is-scroll" style="background-color:white;">
+
     <button class="c-header__icon  c-icon  c-icon--vsmall">{{sprite("menu")}}</button>
 
-    <div class="c-header__logo">
-        <span class="u-hidden-visually">Shopiflight logo</span>
-    </div>
+    <img src="../images/logo-darkbg.png" alt="Shopiflight logo" class="c-header__logo">
```

Comment:
Back to what I said about the logo having to be a physical element. In your case where the logo changes back on the background color, you have two options.

1. Use background-image that is affected by state of your navigation which you done previously (sorry)
2. Use an svg where the text 'light' is green and your `fill` property only affects 'shopif' (would be a cool experiment to try)